### ISSUE TITLE
Python port not working for files in different folders

### ISSUE THREAD

**Issue by shubhamchaudhary** [shubhamchaudhary/universal#7]
The command that I used is :

    ./universal testfiles/hello.c 

Output is as follows:

```shell
chaudhary@localhost:~/code/scripts/universal/python$ ./u testfiles/hello.c 
 = = = = = =   GCC: Compiling testfiles/hello.c file   = = = = = =

Doing:  gcc  -g -O2 -Wall -Wextra -Isrc -rdynamic -fomit-frame-pointer -lm -lrt -o /home/chaudhary/code/scripts/universal/python/testfiles/hello.out testfiles/hello.c
testfiles/hello.c:2:1: warning: return type defaults to ‘int’ [-Wreturn-type]
 main()
 ^
testfiles/hello.c: In function ‘main’:
testfiles/hello.c:5:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
Doing:  ./hello.out
sh: ./hello.out: No such file or directory

```

**Comment by harsimrans** [comment]
Oh ya, broke it in the end while fiddling. Fixed now (commit: 43fbd0e375ea7)

**Comment by harsimrans** [comment]
Fixed it and pushed it to the python-port branch
 Regards
Harsimran Singh 

     On Tuesday, 6 January 2015 5:18 PM, Shubham Chaudhary <notifications@github.com> wrote:
   

 Assigned #7 to @harsimrans.—
